samples/grid/griddemo.cpp

```diff
@@ -920,6 +935,73 @@ void GridFrame::SetColLabelVertAlignment( wxCommandEvent& WXUNUSED(ev) )
 }
 
 
+void GridFrame::SetCornerLabelHorizAlignment( wxCommandEvent& WXUNUSED(ev) )
+{
+    int horiz, vert;
+    grid->GetCornerLabelAlignment( &horiz, &vert );
+
+    switch ( horiz )
+    {
+        case wxALIGN_LEFT:
+            horiz = wxALIGN_CENTRE;
+            break;
+
+        case wxALIGN_CENTRE:
+            horiz = wxALIGN_RIGHT;
+            break;
+
+        case wxALIGN_RIGHT:
+            horiz = wxALIGN_LEFT;
+            break;
+    }
+
+    grid->SetCornerLabelAlignment( horiz, vert );
+}
+
+
+void GridFrame::SetCornerLabelVertAlignment( wxCommandEvent& WXUNUSED(ev) )
+{
+    int horiz, vert;
+    grid->GetCornerLabelAlignment( &horiz, &vert );
+
+    switch ( vert )
+    {
+        case wxALIGN_TOP:
+            vert = wxALIGN_CENTRE;
+            break;
+
+        case wxALIGN_CENTRE:
+            vert = wxALIGN_BOTTOM;
+            break;
+
+        case wxALIGN_BOTTOM:
+            vert = wxALIGN_TOP;
+            break;
+    }
+
+    grid->SetCornerLabelAlignment( horiz, vert );
+}
+
+
+void GridFrame::ToggleCornerLabelOrientation( wxCommandEvent& WXUNUSED(ev) )
```

Comment:
In my testing this doesn't seem to work, i.e. I don't get vertical text in the corner. Is it normal?

Reply:
Damn, sorry, I must have done something wrong as I've retested it now (this was under MSW FWIW) and it does work. Sorry for the false alert.